src/dash/parser/matchers/DurationMatcher.js

```diff
@@ -57,27 +50,16 @@ class DurationMatcher extends BaseMatcher {
 
                 for (let i = 0; i < len; i++) {
                     if (attr.nodeName === attributeList[i]) {
-                        return durationRegex.test(attr.value);
+                        return moment.duration(attr.value, moment.ISO_8601).isValid();
```

Comment:
Could be slightly optimized by _caching_ the result of `moment.duration(attr.value, moment.ISO_8601)` to a variable local to `constructor()` that could be reused in the second arrow function instead of calling `moment.duration(str, moment.ISO_8601)` again.